owner.go

```diff
@@ -1,41 +1,20 @@
 package opslevel
 
-type EntityOwnerGroup struct {
-	Alias string `json:"alias,omitempty" graphql:"groupAlias:alias"`
-	Id    ID     `json:"id"`
-}
-
 type EntityOwnerTeam struct {
 	Alias string `json:"alias,omitempty" graphql:"teamAlias:alias"`
 	Id    ID     `json:"id"`
 }
 
 type EntityOwner struct {
```

Comment:
since this is no longer polymorphic - do we even still need it?

Reply:
I thought about this but had not really looked into how EntityOwner might be used elsewhere or by customers.
If you think it's safe to remove this we can though